blas/unit_test/Test_Blas1_dot.hpp

```diff
@@ -185,26 +163,22 @@ int test_dot() {
   // Test::impl_test_dot<view_type_a_lr, view_type_b_lr, Device>(132231);
 #endif
 
-  // Removing the layout stride test as ViewTypeA a("a", N);
-  // is invalid since the view constructor needs a stride object!
-  /*
-  #if defined(KOKKOSKERNELS_INST_LAYOUTSTRIDE) || \
-      (!defined(KOKKOSKERNELS_ETI_ONLY) &&        \
-       !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS))
-    typedef Kokkos::View<ScalarA*, Kokkos::LayoutStride, Device> view_type_a_ls;
-    typedef Kokkos::View<ScalarB*, Kokkos::LayoutStride, Device> view_type_b_ls;
-    Test::impl_test_dot<view_type_a_ls, view_type_b_ls, Device>(0);
-    Test::impl_test_dot<view_type_a_ls, view_type_b_ls, Device>(13);
-    Test::impl_test_dot<view_type_a_ls, view_type_b_ls, Device>(1024);
-    // Test::impl_test_dot<view_type_a_ls, view_type_b_ls, Device>(132231);
-  #endif
-
-  #if !defined(KOKKOSKERNELS_ETI_ONLY) && \
-      !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS)
-    Test::impl_test_dot<view_type_a_ls, view_type_b_ll, Device>(1024);
-    Test::impl_test_dot<view_type_a_ll, view_type_b_ls, Device>(1024);
-  #endif
-  */
+#if defined(KOKKOSKERNELS_INST_LAYOUTSTRIDE) || \
```

Comment:
I am not sure that we have the ability to actually set `KOKKOSKERNELS_INST_LAYOUTSTRIDE`, at least I do not see it defined in `cmake/kokkoskernels_eti_layouts.cmake`

Reply:
Yeah, there is not testing outside of the `TEST_ETI_ONLY=OFF` for layout stride... I don't know if that's something we want to change or not but since the macro does not exist it seems more prudent to remove it.

I'll have properly look at the rest of the PR

Reply:
Sounds good, I just took those conditions out.